django/chat/models.py

```diff
@@ -69,6 +69,8 @@ class Chat(models.Model):
     # Last access time manually updated when chat is opened
     accessed_at = models.DateTimeField(auto_now_add=True)
 
+    last_message_date = models.DateTimeField(default=timezone.now)
```

Comment:
I don't really think you need this. You could do a query on Message model to get it. Try that and check the query performance in the Django debug toolbar. If performance with the Message query is fine you can just get rid of all changes to this file, the migration, and the merge migration. I'm fine with it either way though - just seems redundant.